scripts/autogen

```diff
@@ -1012,24 +1012,29 @@ def synchronize_backend(
     in_dir, out_dir, dry_run=False, delete=False, no_simplify=False, **kwargs
 ):
     copied = []
+    # Only synchronize sources, but not README.md, Makefile and so on
+    extensions = (".c", ".h", ".i", ".inc", ".S")
     for f in get_files(os.path.join(in_dir, "*")):
-        copied.append(os.path.basename(f))
-        if delete is True:
-            continue
-        if no_simplify is False and f.endswith(".S"):
-            update_via_simpasm(f, out_dir, dry_run=dry_run, **kwargs)
-        else:
-            # Update via copy
-            _, infile = os.path.split(f)
-            outfile_full = os.path.join(out_dir, infile)
-            # The header guards will also be checked later, but if we
-            # don't do it here, the dry-run would fail because of a
-            # mismatching intermediate file
-            if f.endswith(".h"):
-                transform = lambda c: adjust_header_guard_for_filename(c, outfile_full)
+        if f.endswith(extensions):
```

Comment:
Nit: 
```suggestion
        if not f.endswith(extensions):
          continue
```
will keep the code more readable and the diff smaller

Reply:
Done